### PR TITLE
[Task] [OSPP] JAVA implements the native ipmi2 communication protocol

### PR DESCRIPTION
**Pull request by gjjjj0101**
## What's changed?

link https://github.com/apache/hertzbeat/issues/1794

I have successfully implemented the core components of a native IPMI protocol in Java, which includes protocol field declarations, serialization/deserialization, cryptographic algorithms, and abstraction layers. This foundational implementation supports extensive extension of custom IPMI commands. 

Currently, session management, chassis status queries and sensor list info have been implemented. 

image

## Checklist

- [x]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [x]  I have written the necessary doc or comment.
- [x]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

File: collector/collector-basic/src/main/java/org/apache/hertzbeat/collector/collect/ipmi2/protocol/ipmi/security/confidentiality/AseCbc128.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+import org.apache.hertzbeat.collector.collect.ipmi2.client.IpmiSession;
+import org.apache.hertzbeat.collector.collect.ipmi2.utils.ByteConvertUtils;
+import org.apache.hertzbeat.collector.collect.ipmi2.utils.ByteOrderUtils;
+
+/**
+ *  See IPMIv2 Section 13.29
+ */
+public class AseCbc128 extends AbstractIpmiConfidentiality{
```
[the remark this answers is not in the document]
Comment (gjjjj0101) [reply collector/collector-basic/src/main/java/org/apache/hertzbeat/collector/collect/ipmi2/protocol/ipmi/security/confidentiality/AseCbc128.java:37]: thanks, my typo fault

**Comment by zuobiao-zhou** [comment]
Please link the help document in https://github.com/apache/hertzbeat/blob/master/home/sidebars.json

**Comment by tomsun28** [comment]
lgtm!👍 a fantastic job

**Comment by TJxiaobao** [comment]
It was a big task, but you did it very well. Keep it up!👍

**Review by TJxiaobao — APPROVED** [review]
Very well done! 👍

**Pull request merged**